config/notify.yml

```diff
@@ -0,0 +1,11 @@
+development:
+  api_key: <%= ENV['GOVUK_NOTIFY_API_KEY'] %>
+  template_id: <%= ENV['GOVUK_NOTIFY_TEMPLATE_ID'] || "a5583ad3-adab-4c76-b85d-f88bd4a3dc69" %>
+
+test:
+  api_key: <%= ENV['GOVUK_NOTIFY_API_KEY'] %>
+  template_id: <%= ENV['GOVUK_NOTIFY_TEMPLATE_ID'] || "a5583ad3-adab-4c76-b85d-f88bd4a3dc69" %>
+
+production:
+  api_key: <%= ENV['GOVUK_NOTIFY_API_KEY'] %>
+  template_id: <%= ENV['GOVUK_NOTIFY_TEMPLATE_ID'] %>
```

Comment:
I know you've already had a discussion about this so apologies for bringing it back.

If we're controlling these by env vars I don't think we get anything by these being separate ones for different environments - changing env vars changes them all. 

So I'd imagine we'd only want to separate these out if we hard code some attributes for particular environments (which I generally don't think is great since it's not very 12 factor but sometimes pragmatic)

/cc @thomasleese

Reply:
I imagine we might want to hardcode these for the test environment, and perhaps not allow them to be configured with environment variables.

Otherwise, I have no opinion on this, I'm happy with keeping the environments or leaving them out.

Reply:
We may need to put something for the development and test environments directly into the file like some other apps.